src/main/java/com/nhnacademy/marketgg/server/entity/Asset.java

```diff
@@ -16,6 +16,7 @@
 /**
  * 자원 개체입니다.
  *
+ * @author 공통
```

Comment:
@author 공통 보다 다 추가하는 것이 좋을 것 같습니다.
아래는 `RestTemplate` 클래스 공동 저자 작성 예시입니다.

```java
/**
 * Synchronous client to perform HTTP requests, exposing a simple, template
 *
 * (...)
 *
 * which has a more modern API and supports sync, async, and streaming scenarios.
 *
 * @author Arjen Poutsma
 * @author Brian Clozel
 * @author Roy Clarkson
 * @author Juergen Hoeller
 * @author Sam Brannen
 * @author Sebastien Deleuze
 * @since 3.0
 * @see HttpMessageConverter
 */
public class RestTemplate extends InterceptingHttpAccessor implements RestOperations {
    // (...)
}
```

Reply:
월요일 머지데이에 함께하면 좋을 것 같습니다!